.github/script/call-canary-deploy-demo-prod

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+const { spawnOrFail, logger } = require('../../script/cli-utils');
+
+const myArgs = process.argv.splice(2);
+var demoCurrentVersion = myArgs[0];
+
+var npmLatestVersion = spawnOrFail('npm',['view', 'amazon-chime-sdk-js@latest', 'version']);
+var npmBetaVersion = spawnOrFail('npm', ['view', 'amazon-chime-sdk-js@beta', 'version']);
+
+// Removing line breaks from the strings before comparison
+npmLatestVersion = npmLatestVersion.replace(/(\r\n|\n|\r)/gm,'');
+npmBetaVersion = npmBetaVersion.replace(/(\r\n|\n|\r)/gm,'');
+demoCurrentVersion = demoCurrentVersion.replace(/(\r\n|\n|\r)/gm,'');
+
+logger.log('NPM latest version: ', npmLatestVersion);
+logger.log('NPM beta version: ', npmBetaVersion);
+logger.log('Demo current version: ', demoCurrentVersion);
+
+if (npmBetaVersion === npmLatestVersion || npmBetaVersion === '') {
```

Comment:
I think the logic here is a bit convoluted. Can we just do? 
```
// Return 3 for beta 3.0.0-beta.0 and 2.28.0
npmMajorVersion = Math.max(getMajorVersion(npmBetaVersion), getMajorVersion(npmLatestVersion)); 
deployMajorVersion = getMajorVersion(deployVersion);
if (deployMajorVersion === npmMajorVersion) {
      spawnOrFail('script/deploy-canary-demo');
} else if (deployMajorVersion < npmMajorVersion) {
     spawnOrFail('script/deploy-canary-demo', ['-prev']);
} else {
  logger.log('Current version deployment does not match a deployed npm version');
  process.exit(1);
} 
```

Reply:
Ok I had code very similar to this earlier. I will use that.